src/peakDetect.js

```diff
@@ -160,9 +160,10 @@ class PeakDetect {
   }
 
   /**
-   *  onPeak accepts two arguments: a function to call when
-   *  a peak is detected. The value of the peak,
-   *  between 0.0 and 1.0, is passed to the callback.
+   *  onPeak accepts one or two arguments: a callback function to call when
```

Comment:
a callback function to be called ? would be better .

Reply:
A callback function to be called when a peak is detected and a second optinal argument , this function is called with two arguments , first is the current value of peak and second one is the argument that we pass as optional argument to this function. 

@rbtcollins  i have tried to rephrase , i hope we can make smth out it ?

Reply:
Looks great to me

Reply:
would you rephrase it in this commit? then we can have a merge this side

Reply:
@rbtcollins I like your original suggestion here.

@endurance21 think we oughta merge it as is? I'll leave it up to you since you had suggested a change which also sounds good to me.

Reply:
I think we can go with @rbtcollins commit.

Reply:
Thumps up for this.✨